src/app/(proper_react)/(redesign)/Shell/ShellRedesign.tsx

```diff
@@ -0,0 +1,185 @@
+  FaqIcon,
+  ScanInfoIcon,
+  SettingsIcon,
+  TipIcon,
+} from "../../../components/server/Icons";
+import MonitorLogo from "../../images/monitor-logo.svg";
+import styles from "./ShellRedesign.module.scss";
+
+export type Props = {
+  l10n: ExtendedReactLocalization;
+  session: Session;
+  children: ReactNode;
+  countryCode: string;
+  enabledFeatureFlags: FeatureFlagName[];
+  experimentData: ExperimentData["Features"];
+  monthlySubscriptionUrl: string;
+  yearlySubscriptionUrl: string;
+  fxaSettingsUrl: string;
+  subscriptionBillingAmount: {
+    yearly: number;
+    monthly: number;
+  };
+};
+
+export const NavbarList = (props: {
+  l10n: ExtendedReactLocalization;
+  countryCode: string;
+  enabledFeatureFlags: FeatureFlagName[];
+}) => (
+  <ul className={styles.navbarList}>
+    <li key="dashboard">
+      <PageLink
+        href="/user/dashboard"
+        activeClassName={styles.isActive}
+        hasTelemetry={{ link_id: "navigation_dashboard" }}
+      >
+        <DashboardIcon alt="" />
+        {props.l10n.getString("main-nav-link-dashboard-label")}
+      </PageLink>
+    </li>
+    {props.enabledFeatureFlags.includes("EditScanProfileDetails") && (
+      <li key="settings-edit-info">
+        <PageLink
+          href="/user/settings/edit-info"
+          activeClassName={styles.isActive}
+          hasTelemetry={{ link_id: "navigation_edit_info" }}
+        >
+          <ScanInfoIcon alt="" />
+          {props.l10n.getString("settings-tab-label-update-scan-info")}
+        </PageLink>
+      </li>
+    )}
+    <li key="separator-first">
+      <hr />
+    </li>
+    <li key="settings-title">
+      <strong>{props.l10n.getString("main-nav-link-settings-label")}</strong>
+    </li>
+    {!props.enabledFeatureFlags.includes("EditScanProfileDetails") && (
+      <li key="settings-edit-info">
+        <PageLink
+          href="/user/settings/edit-info"
+          activeClassName={styles.isActive}
+          hasTelemetry={{ link_id: "navigation_edit_info" }}
+        >
+          <ScanInfoIcon alt="" />
+          {props.l10n.getString("settings-tab-label-edit-info")}
+        </PageLink>
+      </li>
+    )}
+    <li key="settings-notifications">
+      <PageLink
+        href="/user/settings/notifications"
+        activeClassName={styles.isActive}
+        hasTelemetry={{
+          link_id: "navigation_settings_notifications",
+        }}
+      >
+        <NotificationIcon alt="" />
+        {props.l10n.getString("settings-tab-label-notifications")}
+      </PageLink>
+    </li>
+    <li key="settings-manage-account">
+      <PageLink
+        href="/user/settings/manage-account"
+        activeClassName={styles.isActive}
+        hasTelemetry={{
+          link_id: "navigation_settings_manage_account",
+        }}
+      >
+        <SettingsIcon alt="" />
+        {props.l10n.getString("settings-tab-label-manage-account")}
+      </PageLink>
+    </li>
+    <li key="separator-second">
+      <hr />
+    </li>
```

Comment:
Just thinking about how a screen reader would read this... Would it maybe make more sense to use separate `<ul>`s separated by an `<hr>`?

Reply:
Yeah, I agree that this is not optimal: I’ve split the list into multiple `<ul>`s in [2e3e5b2](https://github.com/mozilla/blurts-server/pull/5671/commits/2e3e5b29f189ed6eaa61a87aca1aec8c925bd0bb).